torch/testing/_internal/common_utils.py

```diff
@@ -79,8 +79,7 @@
 if sys.platform == 'win32':
     FILE_SCHEMA = "file:///"
 
-# Environment variable `IN_CI` is set in `.jenkins/common.sh`.
-IS_IN_CI = os.getenv('IN_CI') == '1'
+IS_CI = os.getenv('CI') == '1'
```

Comment:
Where do we actually set CI to be 1? 

Looking at the test logs, every file should be run with --import-disabled-tests and --import-slow-tests but that's not happening.

Reply:
Can you elaborate? What are you expecting in the logs vs. what you're seeing?